rc/symflower.kak

```diff
@@ -35,14 +35,14 @@ define-command symflower-alternative-file -docstring 'Jump to the alternate file
 		altfile=${kak_buffile%.go}_symflower_test.go
 		test ! -f "$altfile" && echo "fail 'Symflower test file not found'" && exit
 		;;
-	(*.java)
```

Comment:
Can you please comment on the "case" why we want this order so we have it explicitly document.

Reply:
We usually always order automatically according to the literal. Would go with `# REMARK Cases are sorted to match specific extensions first and general language files last.`